public/i18n/en.yaml

```
@@ -910,13 +901,29 @@ no-permissions:
 node-details:
   all: All messages
   information: Information
-  internal-ip: Internal IP
   pod-cidr: Pod CIDRs
-  provider: Provider
   warnings: Warnings
+  region: Region
+  zone: Zone
+  pool: Pool
+  machine-type: Machine Type
+  machine-info:
+    architecture: Architecture
+    provider: Provider
+    internal-ip: Internal IP
+    cpus: CPUs
+    gib: GiB
+    kube-proxy-version: Kube proxy version
+    kubelet-version: Kubelet version
+    memory: Memory
+    operating-system: Operating system
+    pods-capacity: Pods capacity
+    hostname: Hostname
+    title: Machine info
 nodes:
   title: Nodes
   title_details: Node {{nodeName}}
+  description: Kubernetes runs your workload by placing containers into Pods to run on <0>Nodes</0>. A node may be a virtual or physical machine, depending on the cluster. Each node is managed by the control plane and contains the services necessary to run Pods.
```

I rephrased it a bit so it's not exactly the same as on the Kubernetes page. Can you take a look, if it still makes sense? :) 
```suggestion
  description: Kubernetes executes your workload by grouping containers into Pods that operate on <0>Nodes</0>, which can be virtual or physical machines depending on the cluster configuration. The control plane monitors each node, which includes the services required to run Pods.
```